samples/bluetooth/broadcast_audio_sink/src/main.c

```diff
@@ -38,7 +38,7 @@ BUILD_ASSERT(IS_ENABLED(CONFIG_SCAN_SELF) || IS_ENABLED(CONFIG_SCAN_OFFLOAD),
 #endif /* CONFIG_SCAN_SELF */
 
 #define INVALID_BROADCAST_ID        (BT_AUDIO_BROADCAST_ID_MAX + 1)
-#define SYNC_RETRY_COUNT            6 /* similar to retries for connections */
+#define PA_SYNC_INTERVAL_TO_TIMEOUT_RATIO 20 /* Set the timeout relative to interval */
 #define PA_SYNC_SKIP                5
```

Comment:
Will this be enough?

```suggestion
#define PA_SYNC_SKIP                5U
#define PA_SYNC_INTERVAL_TO_TIMEOUT_RATIO (PA_SYNC_SKIP + 6U) /* Set the timeout relative to interval */
```


Controller implementation for review is here:
https://github.com/zephyrproject-rtos/zephyr/blob/ba46cd006c682c6b5214061af12a67666198f85d/subsys/bluetooth/controller/ll_sw/ull_sync.c#L724-L748

Reply:
It's a good question. AFAIK there are no recommendations in the Core Spec about the timeout values for PA sync, but at least to me it makes sense that the timeout is relative to the interval (the longer interval => longer timeout value). 

Whether it should be relative to the sync skip value is more unclear, as it depends entirely on the usecase as well. 

And just FYI, the value `20` was just pulled out of a hat :)